packages/core/content-type-builder/admin/src/components/IconPicker/index.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useRef } from 'react';
+import {
+  Box,
+  Flex,
+  Icon,
+  Typography,
+  Searchbar,
+  IconButton,
+  inputFocusStyle,
+  VisuallyHidden,
+  Field,
+  FieldLabel,
+  FieldInput,
+} from '@strapi/design-system';
+import { Trash, Search } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { getTrad } from '../../utils';
+import COMPONENT_ICONS from '../../utils/componentIcons';
```

Comment:
What do you think about:

1. instead of having this in `utils/` move it into `constants.js`, because it isn't really a utility
2. could you please use a named export? This makes it much easier to grep for those things in the codebase.